js_modules/dagster-ui/packages/ui-core/src/assets/asset-checks/ExecuteChecksButton.tsx

```diff
@@ -63,8 +63,17 @@ export const ExecuteChecksButton = ({
     );
   }
 
-  const jobName = jobNames[0];
-  if (!jobName) {
+  // Ideally all the checks have a common job name, but some older user code versions do not report
+  // job names for checks
+  const commonCheckJobNames = checks.reduce((commonJobNames, check) => {
+    return commonJobNames.filter((name: string) => check.jobNames.includes(name));
+  }, checks[0]?.jobNames || []);
+
+  // If there isn't a common name fallback to the first job name of the associated asset
+  const resolvedJobName =
+    commonCheckJobNames.length > 0 ? commonCheckJobNames[0] : assetJobNames[0];
```

Comment:
its a pretty narrow set of older code versions that dont report job names for checks correctly so i debated just dropping the fallback here